src/components/molecules/DateInput/helpers.ts

```diff
@@ -0,0 +1,120 @@
+import { Constants } from './constants';
+import { ScrollType } from './DateInput.model';
+import { Day } from './DaysGrid/DaysGrid.model';
+import { IBox } from './YearBar/YearBar.model';
+
+export const currentDateToString = (day: number, month: number, year: number) => {
+  const dayString = `${day}`;
```

Comment:
Take a look at Intl.DateTimeFormat. Maybe this will be more flexible for different locales